ga4gh/exceptions.py

```diff
@@ -190,6 +198,84 @@ def getMessage(self):
             self.callSetId, self.variantSetId)
         return message
 
+
+class DataException(BaseServerException):
+    """
+    Exceptions thrown during the server startup, and processing faulty VCFs
+    """
+    message = "Faulty data found or data file is missing."
+
+
+class EmptyDirException(DataException):
+    message = "Diretory empty, no VCF/BCF file was found"
+
+
+class MalFormedException(DataException):
+    """
+    Exception thrown when expected entry or file was missing,
+    or unexpected (meta)data entry was found.
+    """
+    message = "Faulty entry found in data file."
+
+
+class NotIndexedException(MalFormedException):
+    """
+    VCF/BCF files must be indexed, with the same prefix.
+    """
+    def __init__(self, fileName):
+        self.message = (
```

Comment:
Nope, this works @dcolligan. The strings are automatically concatenated and the brackets are just there to make line continuation work (which is the PEP recommended way).